operator/pkg/ciliumendpointslice/ceptocesmap.go

```diff
@@ -167,6 +167,10 @@ func NewCESName(name string) CESName {
 	return CESName(resource.Key{Name: name})
 }
 
+func NewCESNameNamespace(name string, namespace string) CESName {
```

Comment:
Thanks, I changed the name. There are still places, where NewCESName is used and namespace information is not needed (I believe that's the case of cesStore in reconciler). I did not modify it since it didn't concern the double queue.